CONTRIBUTING.md

```diff
@@ -1,7 +1,7 @@
 If you find a bug or you'd like to propose a feature, please feel free to raise
-an issue on our [issue tracker](https://github.com/input-output-hk/cardano-addresses/issues).
+an issue on our [issue tracker](https://github.com/IntersectMBO/cardano-addresses/issues).
 
 Pull requests are welcome!
 
 When creating a pull request, please make sure that your code adheres to our
-[coding standards](https://github.com/input-output-hk/adrestia/blob/master/docs/code/Coding-Standards.md).
+[coding standards](https://github.com/IntersectMBO/adrestia/blob/master/docs/code/Coding-Standards.md).
```

Comment:
Doesn't work — probably the old link should be kept or the file should be copied to this repo?

Reply:
thanks! Fixed